src/include/metrics/metrics_store.h

```diff
@@ -194,11 +195,15 @@ class MetricsStore {
    * Record query execution history
    * @param query_id id of the query
    * @param timestamp time of the query execution
+   * @param param parameter associated with this query
+   * @param oid database oid
    */
-  void RecordQueryTrace(const execution::query_id_t query_id, const uint64_t timestamp) {
+  void RecordQueryTrace(const execution::query_id_t query_id, const uint64_t timestamp,
+                        common::ManagedPointer<const std::vector<parser::ConstantValueExpression>> param,
+                        catalog::db_oid_t oid) {
     NOISEPAGE_ASSERT(ComponentEnabled(MetricsComponent::QUERY_TRACE), "QueryTraceMetric not enabled.");
     NOISEPAGE_ASSERT(query_trace_metric_ != nullptr, "QueryTraceMetric not allocated. Check MetricsStore constructor.");
-    query_trace_metric_->RecordQueryTrace(query_id, timestamp);
+    query_trace_metric_->RecordQueryTrace(query_id, timestamp, param, oid);
```

Comment:
nit: change `oid` to `db_oid` to be more explicit. Also, could you put `db_oid` at the front? It's easier to follow with higher level concept in front followed by decreasing level of concepts.

Reply:
Yeah sure, resolved in latest push.